sycl/test-e2e/Matrix/element_wise_abc.cpp

```diff
@@ -15,4 +15,5 @@
 #define SG_SZ 16
 constexpr size_t TN = 16;
 
+#include "common.hpp"
```

Comment:
In all other matrix tests we include common.hpp as one of the first header files, before we define global variables, so I suggest do the same for this file.